models/remoteworkitem.go

```diff
@@ -0,0 +1,67 @@
+package models
+
+import (
+	"encoding/json"
+)
+
+const (
+	SystemRemoteIssueId = "system.remote_issue_id"
+	SystemTitle         = "system.title"
+	SystemDescription   = "system.description"
+	SystemStatus        = "system.status"
+	ProviderGithub      = "github"
+)
+
+var workItemKeyMaps = map[string]WorkItemMap{
+	ProviderGithub: WorkItemMap{
+		AttributeExpression("title"): SystemTitle,
+		AttributeExpression("body"):  SystemDescription,
+		AttributeExpression("state"): SystemStatus,
+		AttributeExpression("id"):    SystemRemoteIssueId,
+	},
+}
+
+type WorkItemMap map[AttributeExpression]string
+
+// AttributeExpression represents a commonly understood String format for a target path
+type AttributeExpression string
+
+// AttributeAccesor defines the interface between a RemoteWorkItem and the Mapper
+type AttributeAccesor interface {
+	// Get returns the value based on a commonly understood field expression
+	Get(field AttributeExpression) interface{}
+}
+
+// RemoteWorkItem is the Database stored TrackerItem
+type RemoteWorkItem struct {
```

Comment:
This object is technically provided by https://github.com/almighty/almighty-core/pull/111